cluster/juju/layers/kubernetes-master/build/update-addons

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+
+description = """
+Update addon manifests for the charm.
+
+This will clone the kubernetes repo and place the addons in
+<charm>/templates/addons.
+
+Can be run with no arguments and from any folder.
+"""
+
+import argparse
+import os
+import shutil
+import subprocess
+
+os.chdir(os.path.dirname(__file__))
+target = os.path.abspath("../templates/addons")
+
+def sh(cmd):
+    """ Alias for subprocess calls """
+    subprocess.check_call(cmd.split())
+
+def clean_addons_folder():
+    """ Remove and recreate the addons folder """
+    print("Cleaning addons folder")
+    shutil.rmtree(target, ignore_errors=True)
+    os.mkdir(target)
+
+def get_kubernetes_repo():
```

Comment:
I don't like cloning this into the layer directory. Can we use a temporary and a shallow checkout. It's a lot of baggage to suck down.